<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasIdxWdg</class>
 <widget class="QWidget" name="MasIdxWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Indicators</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#0000ff;&quot;&gt;Personal data:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>84</height>
      </size>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>8</y>
        <width>45</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>69</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Activity</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbWeight">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>68</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> kg</string>
      </property>
      <property name="minimum">
       <double>30.000000000000000</double>
      </property>
      <property name="maximum">
       <double>399.990000000000009</double>
      </property>
      <property name="value">
       <double>60.000000000000000</double>
      </property>
     </widget>
     <widget class="QComboBox" name="cbSex">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>4</y>
        <width>68</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>male</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbActivity">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>56</y>
        <width>320</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbHeight">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>28</y>
        <width>68</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> sm</string>
      </property>
      <property name="minimum">
       <number>130</number>
      </property>
      <property name="maximum">
       <number>250</number>
      </property>
      <property name="value">
       <number>170</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>34</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbOld">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>4</y>
        <width>68</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> years</string>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="value">
       <number>35</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>34</y>
        <width>45</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;Calculation:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Indicator</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Body mass index</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Basal metabolic rate (the Mifflin St Jeor equation), kcal/day</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Total heat production, kcal</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Ideal body weight, kg</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <widget class="QPushButton" name="pbClouse">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>4</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
